lib/yabeda/prometheus_patch.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```

Comment:
This file is not needed. We patch yabeda to allow support for a different groupping key when using prometheus push gateways.

Push gateways are a tool for ephemeral workloads (like cron jobs), RMT does not have this facilities well matured yet to incorporate this.